lib/email/notifications.js

```diff
@@ -22,10 +22,10 @@ module.exports = (log, error) => {
 
         let addresses = [], eventType = 'bounced', isDeletionCandidate = false
         if (message.bounce) {
-          addresses = mapBounceComplaintRecipients(message.bounce.bouncedRecipients)
+          addresses = message.bounce.bouncedRecipients
           isDeletionCandidate = true
         } else if (message.complaint) {
-          addresses = mapBounceComplaintRecipients(message.complaint.complainedRecipients)
+          addresses = message.complaint.complainedRecipients
```

Comment:
IIUC, we can expect `complainedRecipients` and `bouncedRecipients` to always be an array of strings now? Where do these notifications originate from?

Reply:
> IIUC, we can expect `complainedRecipients` and `bouncedRecipients` to always be an array of strings now?

They've always been an array of strings since we implemented the outgoing queue in the email service. But that code has never been turned on, and I only wrote this code here last week. It too has never been turned on.

> Where do these notifications originate from?

In this module you should see that `bounced_recipients` and `complained_recipients` are `Vec<EmailAddress>`:

https://github.com/mozilla/fxa-email-service/blob/master/src/queues/notification.rs

The confusion came because the same properties in the SQS notification are `Vec<BouncedRecipient>` and `Vec<ComplainedRecipient>` respectively:

https://github.com/mozilla/fxa-email-service/blob/master/src/queues/sqs/notification/mod.rs

Reply:
Thanks for the explanation, I saw the `Vec<EmailAddress>` in notification.rs but wasn't sure if that was for incoming our outgoing notifications.